.github/workflows/push.yml

```diff
@@ -63,13 +65,17 @@ jobs:
         uses: ./.github/actions/setup-x64
       - name: Test
         uses: ./.github/actions/test-linux
+        env:
+          RUN_CADDY_TESTS: "1"
```

Comment:
I'm not a huge fan of adding env vars for just 2 tests. Can we detect if Caddy is running?

Reply:
What do you think about something like `file_get_contents("https://localhost/serverpush", stream_context_create(["http" => ["timeout" => 1]]);`?

Reply:
That would work for me. We do the same for the DB extensions.